pages/Network_APIs/WorkflowforContribution.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Workflow for Content Production & Contribution
+parent: Content Production & Contribution
+grand_parent: Network APIs
+nav_order: 0
+has_children: false
+---
+
+{: .warning }
+This documentation is currently **under development and subject to change**. It reflects outcomes elaborated by 5G-MAG members as part of **WI: Network Exposure and APIs Supporting Media Services and Applications**
+We welcome and encourage contributions from the broader community. If you are interested in becoming a member of the 5G-MAG and actively participating in shaping this work, please contact the [Project Office](https://www.5g-mag.com/contact)
+
+# Required interactions to exploit network capabilities
+  
+## Pre-conditions
+  * The production company has set up an agreement for Network Capability usages. As result, the production company obtained information, used for authentication. During the process of negotiating the agreement, a set of relevant network capabilities has been identified (e.g. using an API catalogue). As result, the production company is authorized to use certain network capabilities (when available).
+  * The production crew has access to one, or several, Network API Platforms. This platform is accessible through any device/connectivity (e.g. a website accessible through the open internet, or command prompt, a dedicated app, etc …).
+    * Note: For the Network API Platform access, the crew has obtained key access tokens/key/credentials/payment-details in advance.
+  * The production crew have a set of credentials (SIM/eSIM) of the network devices will connect to. The network provides, by default, best-effort connectivity.
+ 
+## Before the Event: 
+ 
+### Step 1) Preparing your device, configuring application clients & servers and configuring client/server flows
+  * Devices have already best effort connectivity .
+  * Through app specific [AppAPI], the client app is properly connected to the Application Server. e.g. Haivision app with BBC server, or LiveU app with LiveU server, etc …
```

Comment:
Note sure about the AppAPI terminology as it may get confused with the network APIs?

I think we may need to distinguish between processes that belong to the application (App + Application Provider) - probably out of scope unless there is need to expose information from such an application to the network and/or viceversa - to be discussed.

Reply:
Interaction between Application client and Application server
"Application-specific API" - enables communication between the production network orchestrator and the production device nodes (e.g. camera control units, vision mixer, wireless microphones, sound mixer, etc.), some of which are UEs.

Check previous page to be consistent with terminology.

Reply:
Should be resolved